app/room.py

```diff
@@ -6,33 +6,34 @@
 import random 
 
 class Room(Storage):
-	"""docstring for Room"""
 	"""
+		docstring for Room
+	
 	The length of this list cannot be more 
 	than the number of total rooms
 	"""
 	List_of_all_members = []
 
-	def __init__(self, Room_name = "",Room_type = ""):
-		self.Room_name = Room_name
-		self.Room_type = Room_type
+	def __init__(self, room_name = "",room_type = ""):
+		self.room_name = room_name
+		self.room_type = room_type
 		
 		#Create a variable that holds the instance of the room type created.
 
-		if self.Room_type == "OFFICE":
-			self.Room_instance =  Office(self.Room_name)
+		if self.room_type == "OFFICE":
+			self.room_instance =  Office(self.room_name)
 
 			Storage.total_no_of_rooms +=6
```

Comment:
Since the end result of this operation will always be an assignment `Storage.total_no_of_rooms =6 ` , you might as well avoid the extra increment then assignment operations and just assign to 6